NGitLab.Mock/Clients/TagClient.cs

```diff
@@ -36,13 +37,14 @@ public Tag Create(TagCreate tag)
             }
         }
 
-        public ReleaseInfo CreateRelease(string name, ReleaseCreate data)
+        [Obsolete("Starting in Gitlab 14 releases cannot be created trough tags. Use `ReleaseClient.Create` instead", false)]
+        public RealeaseInfo CreateRelease(string name, ReleaseCreate data)
         {
             using (Context.BeginOperationScope())
             {
                 var project = GetProject(_projectId, ProjectPermission.Contribute);
                 var tag = project.Repository.CreateReleaseTag(name, data.Description);
-                return new ReleaseInfo
+                return new RealeaseInfo
```

Comment:
Please make sure to merge your branche with main, there were recent changes on the subject.

Reply:
Done, Using ReleaseInfo everywhere